UPGRADING.md

```diff
@@ -0,0 +1,26 @@
+Upgrading iex-ruby-client
+===========================
+
+### Upgrading to >= 0.4.0
+
+#### All errors that return HTTP codes 400-600 result in a IEX::Errors::ClientError exception
+
+On previous versions, calling `IEX::Resources::Chart.get` with an invalid option results on a
```

Comment:
Reading this it's pretty clear that we're now doing something different in various API calls. I've opened https://github.com/dblock/iex-ruby-client/issues/15 so that we can handle errors the same way in all methods (ie. raise IEX::Errors::ClientError on every Faraday::ClientError in all calls. Hopefully we don't need to copy paste that code everywhere and implement it in the base class.